packages/patternfly-4/react-core/src/components/Select/Select.tsx

```diff
@@ -484,6 +488,11 @@ class Select extends React.Component<SelectProps & InjectedOuiaProps, SelectStat
                 <div className={css(styles.selectToggleWrapper)}>
                   {toggleIcon && <span className={css(styles.selectToggleIcon)}>{toggleIcon}</span>}
                   {selections && (Array.isArray(selections) && selections.length > 0) && selectedChips}
+                  {selections && showOnlySelectionCount && (Array.isArray(selections) && selections.length > 0) && (
+                    <div className={css(styles.selectToggleBadge)}>
+                      <span className={css(badgeStyles.badge, badgeStyles.modifiers.read)}>{selections.length}</span>
```

Comment:
Just curious, why did you not do `<Badge isRead>{selections.length}</Badge>` instead of using css classes?